osu.Framework.Font/osu.Framework.Font.csproj

```diff
@@ -32,7 +31,8 @@
     <PackageReference Include="ppy.osu.Framework" Version="2023.228.0" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Resources\**\*" />
+    <!-- commented out since the shaders use a .fs extension, which already seems to be loaded, maybe because of F#, causing build errors... -->
+    <!-- <EmbeddedResource Include="Resources\**\*" /> -->
```

Comment:
I think should revert this change because it's out of the topic?

Reply:
well it doesn't compile if the EmbeddedResources are in the file on my machine (arch)